src/Components/ServiceScene/ServiceScene.tsx

```diff
@@ -278,7 +330,53 @@ export class ServiceScene extends SceneObjectBase<ServiceSceneState> {
     }
   }
 
+  getUrlState() {
```

Comment:
Naive question do we need all this code, if users can't access the beginning service scene?

Reply:
I would hazard a guess you are conflating the `ServiceScene` and `ServiceSelectionScene`, which to be fair, is confusing now that you can select more labels then just service